pkg/nsx/services/vpc/vpc.go

```diff
@@ -204,16 +204,13 @@ func (s *VPCService) ListVPC() []model.Vpc {
 	return vpcSet
 }
 
+// DeleteVPC will try to delete VPC resource from NSX.
 func (s *VPCService) DeleteVPC(path string) error {
```

Comment:
Maybe we can `add skipCacheCheck bool` here to minimize change.

Reply:
in the following logic, when trying to get vpc from cache retuning nil, it will skip the delete cache part, if we add parameter here that will involve changing other logic in service and test which will cause much bigger effort.